SimTracker/SiPhase2Digitizer/python/phase2TrackerDigitizer_cfi.py

```diff
@@ -203,6 +203,11 @@
     )
 )
 
+# Add some additional parameters for 3D pixels
+phase2TrackerDigitizer.Pixel3DDigitizerAlgorithm.NPColumnRadius = cms.double(4.0)
```

Comment:
please do this inside the clone statement that defines `Pixel3DDigitizerAlgorithm` (L87 in this revision)